types/wicket/index.d.ts

```diff
@@ -1,46 +1,48 @@
-declare var Wkt: WktModule.WktStatic;
+export {};
 
-declare namespace WktModule {
-    interface WktStatic {
-        new(obj?: any): Wkt;
-        beginsWith(str: string, sub: string): boolean;
-        endsWith(str: string, sub: string): boolean;
-        delimiter: string;
-        isArray(obj: any): boolean;
-        trim(str: string, sub: string): string;
-        Wkt: typeof Wkt;
-    }
+declare class WktClass {
```

Comment:
It seems like this should just be `export class Wkt { ... }` rather than a declare followed by an `export let Wkt` below since you no longer have something you can reference as a type.